test/input/geo/NWM_2.1_Sample_Datasets/Croton_NY_TEST1/waterbodies/reservoir_outflow_croton_levelpool_warm_nudge_off_12_hours_in.csv

```diff
@@ -0,0 +1,157 @@
+time,inflow,outflow,water_sfc_elev,reservoir_type,reservoir_assimilated_value
```

Comment:
It might help to name this "loop2" to make it obvious that it belongs to the second loop for parity.

Reply:
Changed. Thanks.